crane-cloud/frontend
# chore: add loading and error states on admin sections

# Description

- Add loading states for most admin sections that were showing before data is returned, and error states otherwise!

## Type of change


## Trello Ticket ID

https://trello.com/c/QkOfT6mY

## How Can This Been Tested?

- Run this branch locally and check out most of the admin side sections paying attention to the overview and detail pages

## Checklist:


# Screenshots

![chrome-capture-2023-10-1 (1)](https://github.com/crane-cloud/frontend/assets/63339234/b77415c3-99e1-4970-995b-af16d6e2f7d4)
